src/Controller/AbstractController.php

```diff
@@ -66,7 +66,7 @@ public function progress(): JsonResponse
         $request = $this->getRequest();
 
         /** @var SessionInterface $session */
```

Comment:
Are those typehints still needed?

Reply:
missed the unused namespaces, should be good to go now